docs/RELEASING.md

```diff
@@ -0,0 +1,59 @@
+# Releasing the Connectors project
+
+The Connectors project is packaged as a Gem that is added in the Enterprise Search distribution.
+
+The version scheme we use is **MAJOR.MINOR.PATCH.BUILD** and stored in the `VERSION` file 
```

Comment:
Can link to the `VERSION` file.

Reply:
Addressed in https://github.com/elastic/connectors/pull/72/commits/cc1c4611031af6ee6c2c2be0581f5fc8c722ef86